webpack/assets/javascripts/react_app/common/reduxFormI18n.js

```diff
@@ -0,0 +1,7 @@
+import Validators from 'redux-form-validators';
+// import forEach from 'lodash/forEach';
+
+// extract all default strings in redux-form-validations
+// forEach(Validators.messages, ({ defaultMessage }) => window.n__(defaultMessage));
+
+Validators.formatMessage = ({ defaultMessage }) => __(defaultMessage);
```

Comment:
@waldenraines here is a poor mean solution to the problem.

I didn't address the variable translation, e.g. ideas welcome :)
```»is too long (maximum is {count, number} {count, plural, one {character} other {characters}})«```

Reply:
@ohadlevy that is the same direction I was going in for this as well and I never got the variables to work either.  I'm sure there's a better way but I am not sure what it is.